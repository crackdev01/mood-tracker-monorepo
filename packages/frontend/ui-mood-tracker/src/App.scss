.App {
  text-align: center;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    height: 75px;
    margin-left: 1rem;
    width: auto;
  }
}
