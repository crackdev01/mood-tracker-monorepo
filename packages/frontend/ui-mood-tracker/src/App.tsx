import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import SiteHeader from 'src/components/SiteHeader';

import './App.scss';

function App() {
  return (
    <BrowserRouter>
      <SiteHeader />
    </BrowserRouter>
  );
}

export default App;
