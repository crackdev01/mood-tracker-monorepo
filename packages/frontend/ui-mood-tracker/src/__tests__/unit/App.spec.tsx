import React from 'react';
import { render } from '@testing-library/react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from '../../App';

Enzyme.configure({ adapter: new Adapter() });

jest.mock('../../containers/Home/Home', () => () => 'Home');

describe('App.vue', () => {
  test('renders', () => {
    const w = shallow(<App />);
    expect(w.debug()).toMatchSnapshot();
  });
});
