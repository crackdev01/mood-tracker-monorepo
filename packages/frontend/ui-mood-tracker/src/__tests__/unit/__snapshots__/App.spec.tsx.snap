// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
"<Provider store={{...}}>
  <PersistGate loading={{...}} persistor={{...}}>
    <Home />
  </PersistGate>
</Provider>"
`;
